PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java

```diff
@@ -128,10 +128,14 @@ public void onItemClick(AdapterView<?> arg0, View view,
 								SessionHistory.currSessionID = 1;
 							}
 							updatePoints(position);
-							getmDbHandler().setCompletedScenario(
-									scene.getScenarioName());
+							Intent intent = new Intent(GameActivity.this, DisplayPointsActivity.class);
```

Comment:
If you can give the intent a name that also informs about the activity it leads to, it will be great